nifi-nar-bundles/nifi-graph-bundle/nifi-graph-processors/src/main/java/org/apache/nifi/processors/graph/ExecuteGraphQueryRecord.java

```diff
@@ -240,6 +259,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             delta = (end - start) / 1000;
             if (getLogger().isDebugEnabled()){
                 getLogger().debug(String.format("Took %s seconds.", delta));
+                getLogger().debug(String.format("Handled %d records", records));
```

Comment:
This could be done in one statement (and doesn't need the isDebugEnabled check IIRC)

Reply:
I prefer to put the allocation inside of that check, but yeah, it could be one line.